redirectPostLogin.php

```diff
@@ -0,0 +1,79 @@
+<html>
+    <head>
+        <title>Member</title>
+        		<script>
+	function gotologin() {
+    location.href = "login.php";
+}
+
+</script>
+    </head>
+    <body bgcolor=#b2b2b2>
+        <div align="center" style="background-color: whitesmoke">
+            <?php
+            session_start();
+            require_once 'connect.php';
+
+//3. If the form is submitted or not.
+//3.1 If the form is submitted
+            if (isset($_POST['username']) and isset($_POST['password'])) {
+//3.1.1 Assigning posted values to variables.
+                $username = $_POST['username'];
+                $password = $_POST['password'];
+
+//3.1.2 Checking the values are existing in the database or not
+                $query_userType = "SELECT user_type FROM INFO WHERE username='$username' and password='$password'";
+                $userType = mysqli_query($con, $query_userType) or die(mysqli_error());
+                $row = mysqli_fetch_assoc($userType);
+                
+                $count = mysqli_num_rows($userType);
+                if ($count == 1) {
+                    $_SESSION['username'] = $username;
+                    $_SESSION['userType'] = $row['user_type'];
+                } else {
+//3.1.3 If the login credentials doesn't match, he will be shown with an error message.
+                    echo "Invalid Login Credentials.";
+                }
+            }

```

Comment:
I think for this section, we don't need to check the user type, it makes more sense to throw an error for when we're actually using it. Like below when you're checking type. We can have an `else` statement for `Unknown user type`. 

This way we can shorten this section quite a bit I think. Into:
```php
$login_query = "SELECT username, password, user_type" 
                         ."FROM INFO"
                         ." WHERE username='$username' and password='`$password'";
$login_result = mysqli_query($con, $login_query) or die(mysqli_error());
if(mysqli_num_rows($login_result) < 1){
    echo "Invalid Login Credentials"l
} else {
    $login_row = mysqli_fetch_assoc($login_result);
    $_SESSION['username'] = $login_row('username');
    $_SESSION['user_type'] = $login_row('user_type');
}
```